### DISCUSSION TITLE
is Homebrew going to participate in GSoC 2023?

### DISCUSSION THREAD

**Discussion by YesShubham01** [Homebrew/discussions#4083] [Getting started]
### Output of `brew config`

```shell
.
```

### Output of `brew doctor`

```shell
.
```

### Description of issue

Hello Everyone,
I am interested in working on Homebrew. I want to participate in GSoC. So i wanted to ask if Homebrew is going to participate in Gsoc this year or not? 
and also i need help with installation.

**Comment by SMillerDev** [comment]
https://github.com/homebrew/gsoc has all the information about GSoC from Homebrew.
